### DISCUSSION TITLE
Upload from local subfolder to remote root

### DISCUSSION THREAD

**Discussion by glatocha** [Natizyskunk/vscode-sftp#400] [Q&A] [answered]
Hi,
I have a project which is built into dist folder. When I try to upload it to my FTP it is creating the dist folder on the server. Even if I only upload a single file. 
How to configure that?

Thanks in advance

**Comment by satiromarra** [comment]
@glatocha Is the `dist` directory the root of your remote project? like 'public_html' or 'httpdocs'.

**Reply by glatocha** [reply]
Yes. It's a subdomain so it has a different name but yes. That's the folder where the index.html sits

**Reply by satiromarra** [reply, marked as answer]
Then you must add to the connection:
```json
"context":"dist",
"remotePath":"your-remote-public",
```

**Reply by glatocha** [reply]
Yes, perfect. Much obliged
